Module.php

```diff
@@ -31,7 +31,7 @@ class Module extends \yii\base\Module
     /**
      * @var bool If true, users are required to enter a username
      */
-    public $requireUsername = false;
+    public $requireUsername = true;
```

Comment:
This probably slipped into your commit, I don't think the author will want to default to a required username.